README.rst

```diff
@@ -38,7 +38,7 @@ To install this package simply use pip::
 Support / Feedback
 ==================
 
-For questions, discussions, or general technical support, visit the MongoDB Community Forums.
+For questions, discussions, or general technical support, visit the `MongoDB Community Forums <https://developer.mongodb.com/community/forums/tag/python>`_.
```

Comment:
Please add a link to this GitHub repository in the reference to 'this GitHub repository' on L48

Reply:
Done.